app/managers.py

```diff
@@ -0,0 +1,42 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
+
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO actors (first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+
+    def all(self):
+        actor_cursor = self._connection.execute(
+            f"SELECT * FROM actors"
+        )
+        return [Actor(*row) for row in actor_cursor]
+
+
+    def update(self, id_to_update: int, new_first_name: str, new_last_name: str ) -> None:
+        self._connection.execute(
+            f"UPDATE actors "
+            "SET {new_first_name} = ? "
+            "SET {new_last_name} = ? "
```

Comment:
The SQL syntax in the `update` method is incorrect. You should use a single `SET` clause with multiple columns, like `SET first_name = ?, last_name = ?`.